Brewfile

```diff
@@ -0,0 +1,13 @@
+# Brewfile: a Gemfile, but for Homebrew
+#
+# Install the dependencies in this file by running:
+# brew bundle
+#
+# Read more about Brewfiles here:
+# https://thoughtbot.com/blog/brewfile-a-gemfile-but-for-homebrew
+
+# required by Rails Active Record
+brew 'postgres'
+
+# required by gem 'mimemagic' v0.3.9
+brew 'shared-mime-info'
```

Comment:
This file isn't needed. The upgrade to rails uses another gem and stops using mimemagic